cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -126,7 +127,7 @@ func TestEvaluateExecute(t *testing.T) {
 			},
 
 			ExpectedOutputValidate: func(t *testing.T, output string, resultPath string) {
-				assert.Contains(t, output, `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=14, coverage-statement=10, files-executed=1, response-no-error=1, response-no-excess=1, response-with-code=1`)
+				assert.Regexp(t, `.*Evaluation score for "symflower/symbolic-execution" \("code-no-excess"\): score=14, coverage-statement=10, files-executed=1, processing-time=\d+, response-no-error=1, response-no-excess=1, response-with-code=1.*`, output)
```

Comment:
why .* at the end? what should come after that? Isn't that the only line?

Reply:
And start.

Reply:
Was able to remove both. Guess it's just about that the match exists, not that it needs to be a complete match.